code/__DEFINES/~russ_defines/lowpop.dm

```diff
@@ -0,0 +1 @@
+#define LOWPOP_THRESHOLD 8 // how many players are needed to disable egalitarian mode
```

Comment:
Could this be a configuration option instead of a macro? Besides being a little easier to find, it being a config would allow an admin to mess around with the setting while the server is up. As a macro, it'd require recompilation to change it.